test/Microsoft.Health.Fhir.Ingest.UnitTests/Telemetry/FhirExceptionTelemetryProcessorTests.cs

```diff
@@ -17,14 +17,14 @@ namespace Microsoft.Health.Fhir.Ingest.Telemetry
     public class FhirExceptionTelemetryProcessorTests
     {
         [Theory]
-        [InlineData(typeof(MultipleResourceFoundException<object>))]
-        [InlineData(typeof(PatientDeviceMismatchException))]
-        [InlineData(typeof(NotSupportedException))]
-        [InlineData(typeof(FhirResourceNotFoundException))]
-        [InlineData(typeof(ResourceIdentityNotDefinedException))]
-        [InlineData(typeof(TemplateNotFoundException))]
-        [InlineData(typeof(InvalidQuantityFhirValueException))]
-        public void GivenHandledExceptionTypes_WhenHandleExpection_ThenMetricLoggedAndTrueReturned_Test(System.Type exType)
+        [InlineData(typeof(MultipleResourceFoundException<object>), "MultipleObjectFoundException")]
+        [InlineData(typeof(PatientDeviceMismatchException), nameof(PatientDeviceMismatchException))]
+        [InlineData(typeof(NotSupportedException), nameof(NotSupportedException))]
+        [InlineData(typeof(FhirResourceNotFoundException), "DeviceNotFoundException")]
+        [InlineData(typeof(ResourceIdentityNotDefinedException), "DeviceIdentityNotDefinedException")]
+        [InlineData(typeof(TemplateNotFoundException), nameof(TemplateNotFoundException))]
+        [InlineData(typeof(InvalidQuantityFhirValueException), nameof(InvalidQuantityFhirValueException))]
+        public void GivenHandledExceptionTypes_WhenHandleExpection_ThenMetricLoggedAndTrueReturned_Test(System.Type exType, string metricName)
```

Comment:
nit: Update all the test names in `FhirExceptionTelemetryProcessorTests` and `NormalizationExceptionTelemetryProcessorTests` now that what's returned has changed?